pulsar-common/src/main/resources/findbugsExclude.xml

```diff
@@ -0,0 +1,81 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<FindBugsFilter>
+    <Match>
+        <Class name="org.apache.pulsar.PulsarVersion"/>
+    </Match>
+
+    <Match>
+        <Class name="~org\.apache\.pulsar\.shaded\.com\.google\.protobuf.*"/>
+    </Match>
+
+    <Match>
+        <Class name="~org\.apache\.pulsar\.common\.api\.proto.*"/>
+    </Match>
+
+    <Match>
+        <Class name="org.apache.pulsar.common.nar.FileUtils"/>
+        <Method name="deleteFilesInDirectory"/>
+        <Bug pattern="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE"/>
+    </Match>
+
+    <Match>
+        <Class name="org.apache.pulsar.common.policies.data.FunctionStats$FunctionInstanceStats$FunctionInstanceStatsData"/>
+        <Bug pattern="EQ_OVERRIDING_EQUALS_NOT_SYMMETRIC"/>
+    </Match>
+
+    <Match>
+        <Class name="org.apache.pulsar.common.util.collections.ConcurrentBitSet"/>
+        <Bug pattern="EQ_DOESNT_OVERRIDE_EQUALS"/>
+    </Match>
+
+    <Match>
+        <Class name="org.apache.pulsar.common.util.collections.ConcurrentBitSetRecyclable"/>
+        <Bug pattern="EQ_DOESNT_OVERRIDE_EQUALS"/>
+    </Match>
+
+    <Match>
+        <Class name="org.apache.pulsar.common.util.NamespaceBundleStatsComparator"/>
+        <Bug pattern="SE_COMPARATOR_SHOULD_BE_SERIALIZABLE"/>
+    </Match>
+
+    <Match>
```

Comment:
Why do we exclude the following bugs? At a minimum, we should only exclude those bugs for a specific class if we have a clear idea that we should exclude them.

Reply:
There are some error will change a lot of classes file, for example, the `URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD`, most of this error is found in the stats classes. We have a lot of classes that use the stats class so it will change across multiple files. Can I create an issue for that and use other PRs for fixing it?

Reply:
I clean most of the bugs. But there has still some bugs marked excluded by me. Because it will cause almost 130 bugs and it will need to change a lot of files across different modules. 
Can I fix them in other PRs?

Reply:
Sure. Can you create a follow-up issue and add a comment with the issue link to the file?

Reply:
issue created #9080